components/dictionary.txt

```diff
@@ -100,13 +124,16 @@ pre
 Protip
 README
 renv
-Repo
+repo
 reproducibility
 rerio
+rint
 Rittenhouse
 Riverol
 Rohan
+Rscript
```

Comment:
I would expect we usually want this styled as `Rscript` so maybe it isn't needed here?